.ci/gpg/create-keyring.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
I could not find the changes in the release doc to know the steps that should be performed now.

Reply:
This file is run by CI only. You don't need to run this file yourself at all. See the changes to `docs/contribution-guidelines/releasing.md`.

Reply:
It is hidden. Tks.